### DISCUSSION TITLE
能否将输入法状态提示的显示位置设置在屏幕中央？

### DISCUSSION THREAD

**Discussion by wha4up** [shuax/MouseInc.Settings#407] [Ideas]
或者增加相关设置界面。

**Comment by kevzcr** [comment]
> 或者增加相关设置界面。
要是能象CapsLock键提醒那样反应灵敏，可以象按键回显那样进行位置等相关设置就好了。

**Comment by sumeco** [comment]
你可以安装ModernFlyouts试试看。ModernFlyouts+mouseinc完美

**Reply by wha4up** [reply]
谢谢分享，不过我一般都是在用办公电脑，资源相当紧张，开个office都卡半天，所以为了这一个需求再装款软件就有点得不偿失了。
